proxy/src/main/java/com/velocitypowered/proxy/connection/MinecraftConnection.java

```diff
@@ -164,6 +165,9 @@ public EventLoop eventLoop() {
    * @param msg the message to write
    */
   public void write(Object msg) {
+    if (msg instanceof BossBar && protocolVersion.getProtocol() < ProtocolVersion.MINECRAFT_1_9.getProtocol()) { // just to be sure
```

Comment:
A valid version isn't enforced for `AvailableCommands`, another packet not supported on earlier versions. Why should boss bars have version enforced?

Reply:
When i tested it on bungee level with my api, it looks like when you send a bossbar packet on 1.8 for example, the player gets disconnected. This behaviour is not seen on the commands and brigadier, thats why its good to check.